src/Hocon.API.Tests/HoconAPISpec.ApproveConfiguration.approved.txt

```diff
@@ -7,30 +7,29 @@ namespace Hocon
+    public class Config : Hocon.HoconObject, System.IEquatable<Hocon.Config>, System.Runtime.Serialization.ISerializable
     {
-        [System.ObsoleteAttribute("For json serialization/deserialization only", true)]
-        protected Config() { }
-        protected Config(Hocon.HoconValue value) { }
-        protected Config(Hocon.HoconValue value, Hocon.Config fallback) { }
-        public Config(Hocon.HoconRoot root) { }
-        public Config(Hocon.HoconRoot root, Hocon.Config fallback) { }
+        public Config(Hocon.HoconElement root) { }
         [System.ObsoleteAttribute("Used for serialization only", true)]
-        public Config(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
+        protected Config(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
         public static Hocon.Config Empty { get; }
-        public virtual System.Collections.Generic.IReadOnlyList<Hocon.HoconValue> Fallbacks { get; }
+        public virtual System.Collections.Generic.IReadOnlyList<Hocon.HoconObject> Fallbacks { get; }
         public virtual bool IsEmpty { get; }
-        public Hocon.HoconValue Root { get; }
-        protected System.Collections.Generic.List<Hocon.HoconValue> _fallbacks { get; }
-        public override System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, Hocon.HoconField>> AsEnumerable() { }
+        public Hocon.HoconObject Root { get; }
+        protected System.Collections.Generic.List<Hocon.HoconObject> _fallbacks { get; }
+        public override System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, Hocon.HoconElement>> AsEnumerable() { }
+        public static Hocon.Config Create(Hocon.HoconElement root) { }
+        public static Hocon.Config Deserialize(string raw) { }
+        public override bool Equals(Hocon.HoconElement other) { }
         public virtual bool Equals(Hocon.Config other) { }
-        public override bool Equals(object obj) { }
         public virtual Hocon.Config GetConfig(string path) { }
         public virtual Hocon.Config GetConfig(Hocon.HoconPath path) { }
-        protected override Hocon.HoconValue GetNode(Hocon.HoconPath path) { }
+        [System.ObsoleteAttribute("Used for serialization only", true)]
         public void GetObjectData(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
+        public override Hocon.HoconElement GetValue(Hocon.HoconPath path) { }
+        public string Serialize() { }
         public string ToString(bool useFallbackValues) { }
-        protected override bool TryGetNode(Hocon.HoconPath path, out Hocon.HoconValue result) { }
+        public override bool TryGetValue(Hocon.HoconPath path, out Hocon.HoconElement result) { }
```

Comment:
Same as above - what's the significance of this change too?

Reply:
Same as the above, there is no `HoconValue` class.